diff --git a/node_modules/@ghostery/adblocker/dist/commonjs/engine/engine.d.ts b/node_modules/@ghostery/adblocker/dist/commonjs/engine/engine.d.ts
index 13e2633..77ce6f4 100644
--- a/node_modules/@ghostery/adblocker/dist/commonjs/engine/engine.d.ts
+++ b/node_modules/@ghostery/adblocker/dist/commonjs/engine/engine.d.ts
@@ -25,6 +25,7 @@ export declare const ENGINE_VERSION = 731;
 export interface BlockingResponse {
     match: boolean;
     redirect: undefined | {
+        name: string;
         body: string;
         contentType: string;
         dataUrl: string;
diff --git a/node_modules/@ghostery/adblocker/dist/commonjs/resources.d.ts b/node_modules/@ghostery/adblocker/dist/commonjs/resources.d.ts
index 0c037a9..0020c9b 100644
--- a/node_modules/@ghostery/adblocker/dist/commonjs/resources.d.ts
+++ b/node_modules/@ghostery/adblocker/dist/commonjs/resources.d.ts
@@ -43,6 +43,7 @@ export default class Resources {
      */
     updateAliases(): void;
     getResource(name: string): {
+        name: string;
         body: string;
         contentType: string;
         dataUrl: string;
diff --git a/node_modules/@ghostery/adblocker/dist/commonjs/resources.js b/node_modules/@ghostery/adblocker/dist/commonjs/resources.js
index b877949..f616c3b 100644
--- a/node_modules/@ghostery/adblocker/dist/commonjs/resources.js
+++ b/node_modules/@ghostery/adblocker/dist/commonjs/resources.js
@@ -229,7 +229,11 @@ class Resources {
         }
     }
     getResource(name) {
-        const { body, contentType } = this.resourcesByName.get(name) || (0, small_1.getResourceForMime)(name);
+        const resource = this.resourcesByName.get(name) || getResourceForMime(name);
+        const { contentType, body } = resource;
+        if ('name' in resource) {
+            name = resource.name;
+        }
         let dataUrl;
         if (contentType.indexOf(';') !== -1) {
             dataUrl = `data:${contentType},${body}`;
diff --git a/node_modules/@ghostery/adblocker/dist/esm/engine/engine.d.ts b/node_modules/@ghostery/adblocker/dist/esm/engine/engine.d.ts
index 13e2633..77ce6f4 100644
--- a/node_modules/@ghostery/adblocker/dist/esm/engine/engine.d.ts
+++ b/node_modules/@ghostery/adblocker/dist/esm/engine/engine.d.ts
@@ -25,6 +25,7 @@ export declare const ENGINE_VERSION = 731;
 export interface BlockingResponse {
     match: boolean;
     redirect: undefined | {
+        name: string;
         body: string;
         contentType: string;
         dataUrl: string;
diff --git a/node_modules/@ghostery/adblocker/dist/esm/resources.d.ts b/node_modules/@ghostery/adblocker/dist/esm/resources.d.ts
index 0c037a9..0020c9b 100644
--- a/node_modules/@ghostery/adblocker/dist/esm/resources.d.ts
+++ b/node_modules/@ghostery/adblocker/dist/esm/resources.d.ts
@@ -43,6 +43,7 @@ export default class Resources {
      */
     updateAliases(): void;
     getResource(name: string): {
+        name: string;
         body: string;
         contentType: string;
         dataUrl: string;
diff --git a/node_modules/@ghostery/adblocker/dist/esm/resources.js b/node_modules/@ghostery/adblocker/dist/esm/resources.js
index b9772dd..4aeb14c 100644
--- a/node_modules/@ghostery/adblocker/dist/esm/resources.js
+++ b/node_modules/@ghostery/adblocker/dist/esm/resources.js
@@ -227,7 +227,11 @@ export default class Resources {
         }
     }
     getResource(name) {
-        const { body, contentType } = this.resourcesByName.get(name) || getResourceForMime(name);
+        const resource = this.resourcesByName.get(name) || getResourceForMime(name);
+        const { contentType, body } = resource;
+        if ('name' in resource) {
+            name = resource.name;
+        }
         let dataUrl;
         if (contentType.indexOf(';') !== -1) {
             dataUrl = `data:${contentType},${body}`;
