diff --git a/node_modules/rollup/dist/es/shared/node-entry.js b/node_modules/rollup/dist/es/shared/node-entry.js
index 3533549..4f72e0c 100644
--- a/node_modules/rollup/dist/es/shared/node-entry.js
+++ b/node_modules/rollup/dist/es/shared/node-entry.js
@@ -16408,7 +16408,7 @@ class Chunk {
                 : relative(this.inputBase, idWithoutExtension);
         }
         else {
-            return `_virtual/${basename(idWithoutExtension)}`;
+            return `virtual/${basename(idWithoutExtension)}`;
         }
     }
     getReexportSpecifiers() {
diff --git a/node_modules/rollup/dist/shared/rollup.js b/node_modules/rollup/dist/shared/rollup.js
index 204e8a7..4dbce45 100644
--- a/node_modules/rollup/dist/shared/rollup.js
+++ b/node_modules/rollup/dist/shared/rollup.js
@@ -17795,7 +17795,7 @@ class Chunk {
                 : relative(this.inputBase, idWithoutExtension);
         }
         else {
-            return `_virtual/${node_path.basename(idWithoutExtension)}`;
+            return `virtual/${node_path.basename(idWithoutExtension)}`;
         }
     }
     getReexportSpecifiers() {
