diff --git a/node_modules/vite/dist/node/chunks/dep-5e7f419b.js b/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
index ff6c927..44c765f 100644
--- a/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
+++ b/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
@@ -43780,7 +43780,6 @@ function resolveChokidarOptions(config, options) {
     const resolvedWatchOptions = {
         ignored: [
             '**/.git/**',
-            '**/node_modules/**',
             '**/test-results/**',
             out.escapePath(config.cacheDir) + '/**',
             ...(Array.isArray(ignored) ? ignored : [ignored]),
