diff --git a/node_modules/vite/dist/node/chunks/dep-bb8a8339.js b/node_modules/vite/dist/node/chunks/dep-bb8a8339.js
index 417b6a1..2a7e0f2 100644
--- a/node_modules/vite/dist/node/chunks/dep-bb8a8339.js
+++ b/node_modules/vite/dist/node/chunks/dep-bb8a8339.js
@@ -37775,8 +37775,6 @@ function buildHtmlPlugin(config) {
                     const importee = bundle[file];
                     if (importee?.type === 'chunk' && !seen.has(file)) {
                         seen.add(file);
-                        // post-order traversal
-                        chunks.push(...getImportedChunks(importee, seen));
                         chunks.push(importee);
                     }
                 });
@@ -41165,7 +41163,7 @@ async function transformGlobImport(code, id, root, resolveId, isProduction, rest
         return staticImports;
     }))).flat();
     if (staticImports.length)
-        s.prepend(`${staticImports.join(';')};`);
+        s.append(`${staticImports.join(';')};`);
     return {
         s,
         matches,
@@ -47101,7 +47099,6 @@ function resolveChokidarOptions(config, options) {
     const resolvedWatchOptions = {
         ignored: [
             '**/.git/**',
-            '**/node_modules/**',
             '**/test-results/**',
             glob.escapePath(config.cacheDir) + '/**',
             ...(Array.isArray(ignored) ? ignored : [ignored]),
