diff --git a/node_modules/vite/dist/node/chunks/dep-5e7f419b.js b/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
index ff6c927..0f4b3f8 100644
--- a/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
+++ b/node_modules/vite/dist/node/chunks/dep-5e7f419b.js
@@ -37895,7 +37895,7 @@ async function transformGlobImport(code, id, root, resolveId, isProduction, rest
         return staticImports;
     }))).flat();
     if (staticImports.length)
-        s.prepend(`${staticImports.join(';')};`);
+        s.append(`${staticImports.join(';')};`);
     return {
         s,
         matches,
@@ -43780,7 +43780,6 @@ function resolveChokidarOptions(config, options) {
     const resolvedWatchOptions = {
         ignored: [
             '**/.git/**',
-            '**/node_modules/**',
             '**/test-results/**',
             out.escapePath(config.cacheDir) + '/**',
             ...(Array.isArray(ignored) ? ignored : [ignored]),
