diff --git a/node_modules/web-ext/lib/util/file-filter.js b/node_modules/web-ext/lib/util/file-filter.js
index 0fe23ce..960c57e 100644
--- a/node_modules/web-ext/lib/util/file-filter.js
+++ b/node_modules/web-ext/lib/util/file-filter.js
@@ -25,9 +25,7 @@ export class FileFilter {
   constructor({
     baseIgnoredPatterns = ['**/*.xpi', '**/*.zip', '**/.*',
     // any hidden file and folder
-    '**/.*/**/*',
-    // and the content inside hidden folder
-    '**/node_modules', '**/node_modules/**/*'],
+    '**/.*/**/*'],
     ignoreFiles = [],
     sourceDir,
     artifactsDir
